### ISSUE TITLE
error-reporting: Errors not sent when providing an API key only

### ISSUE THREAD

**Issue by DominicKramer** [GoogleCloudPlatform/google-cloud-node#2302]
- I ran `export NODE_ENV=production`
- I installed express and cloud-errors.
- I run the following code:

I could not see errors in the Error Reporting UI for this project.
I could successfully send manually errors to this project with this API key (by calling the API from cURL).

```
var express = require('express');
var app = express();
var errorHandler = require('@google/cloud-errors')({
  projectId: "<my project>",
  key: "<my api key>",
  serviceContext: {
      service: 'steren-test-node',
      version: 'test1'
  }
});

app.get(
  '/errorRoute',
  function ( req, res, next ) {
    // You can push in errors manually
    res.send("Error");
    res.end();
    next(new Error("Got traffic on the errorRoute"));
  }
);

app.get(
  '/anotherRoute',
  function ( req, res, next ) {
    // It'll even log potentially unexpected errors
    JSON.parse("{\"malformedJson\": true");
  }
)

// Just use the express plugin
app.use(errorHandler.express);

app.listen(
  3000
  , function ( ) {
    console.log('Server has been started on port 3000');
  }
);
```

This issue has been migrated from the cloud-errors-nodejs issue [#10](https://github.com/GoogleCloudPlatform/cloud-errors-nodejs/issues/10).

**Comment by ivanseidel** [comment]
Same problem here. This is really bad, because it seems it's not even attempting to report errors (watched requests from Charles, and none was to `https://clouderrorreporting.clients6.google.com/v1beta1/projects/<myproj>/services?key=...`.

It seems that it's validating OAuth before sending, but not being able to "check" the token, because there isn't none.

**Comment by stephenplusplus** [comment]
Sorry, handing it over to @ofrobots who oversees the logging-* and error-reporting.

**Comment by kjin** [comment]
@ivanseidel, and @steren who filed the original bug

This is fixed in error-reporting v0.2.2, please re-open this bug if you have any issues with it!
